packages/babel-generator/test/fixtures/class-brand-check/output.js

```diff
@@ -0,0 +1,11 @@
+class Range {
+  equals(range) {
+      /* 1 */
+
+      /* 2 */
+
+      /* 3 */
```

Comment:
The output here is incorrect. It seems to me the comments are registered as `innerComments` of the BodyStatement, it should be attached as the `innerComments` of `ClassHasInstanceExpression` / `MetaProperty`.